org.eclipse.winery.repository.ui/src/app/instance/instance.service.ts

```diff
@@ -11,15 +11,15 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
  ********************************************************************************/
-import { Injectable } from '@angular/core';
-import { Headers, RequestOptions } from '@angular/http';
-import { Observable } from 'rxjs';
-import { backendBaseURL } from '../configuration';
-import { WineryInstance, WineryTopologyTemplate } from '../model/wineryComponent';
-import { ToscaComponent } from '../model/toscaComponent';
-import { ToscaTypes } from '../model/enums';
-import { WineryVersion } from '../model/wineryVersion';
-import { HttpClient, HttpResponse } from '@angular/common/http';
+import {Injectable} from '@angular/core';
+import {Observable} from 'rxjs';
```

Comment:
Please reformat this file - after importing the code style.

Reply:
Done